### PR TITLE
Add option for enabling thread support

### PR DESCRIPTION
**Pull request by otsukay**
Hi, this PR adds option for thread support to fujitsu-fftw.

### CODE REVIEW

File: var/spack/repos/builtin/packages/fujitsu-fftw/package.py [hunk 1]
```diff
@@ -33,10 +33,12 @@ class FujitsuFftw(FftwBase):
     homepage = "https://github.com/fujitsu/fftw3"
     url = "https://github.com/fujitsu/fftw3/archive/sve-v1.0.0.tar.gz"
 
+    version('1.1.0', sha256='47b01a20846802041a9533a115f816b973cc9b15b3e827a2f0caffaae34a6c9d')
     version('1.0.0', sha256='b5931e352355d8d1ffeb215922f4b96de11b8585c423fceeaffbf3d5436f6f2f')
 
     variant('shared', default=True, description='Builds a shared version of the library')
     variant('openmp', default=True, description="Enable OpenMP support")
+    variant('threads', default=True, description="Enable thread support")
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/fujitsu-fftw/package.py]: How do `openmp` and `threads` play with each other? Is it possible to have `~threads+openmp`?

**Review by alalazo — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/fujitsu-fftw/package.py (hunk 1 above)
Comment (alalazo) [reply var/spack/repos/builtin/packages/fujitsu-fftw/package.py]: @otsukay

Comment (otsukay) [reply var/spack/repos/builtin/packages/fujitsu-fftw/package.py]: @alalazo  I am very sorry that I forgot to reply to your comment, and thank you for the reminder. 
These two options, `opnemp` and `threads`, can be specified independently. I confirmed that it is possible to build the code in four ways `{+,~}openmp{+,~}threads`.

Comment (alalazo) [reply var/spack/repos/builtin/packages/fujitsu-fftw/package.py]: No worries, thanks for the reply!

**Comment by alalazo** [comment]
@spackbot fix style

**Review by alalazo — APPROVED** [review]

**Pull request merged**